doc/changes.rst

```diff
@@ -5,7 +5,7 @@ desitarget Change Log
 2.6.2 (unreleased)
 ------------------
 
-* Reproducible output from :func:`io.read_targets_in_hp()` [`PR #806`_].
```

Comment:
is it intended that the line for PR806 is removed here?

Reply:
Nice catch! This was a mistake on my part. I added this line back into the `changes.rst` file.